testing/communication/ParameterAssignmentShape.ttl

```diff
@@ -19,33 +19,3 @@ shapes:
 		sh:prefix "ids" ;
 		sh:namespace "https://w3id.org/idsa/core/"^^xsd:anyURI ;
 	] .
-
```

Comment:
This entire file can now be deleted, since it does not add any benefit anymore. Thank you!

Reply:
DONE.